cgo/lapack.go

```diff
@@ -497,6 +498,83 @@ func (impl Implementation) Dgels(trans blas.Transpose, m, n, nrhs int, a []float
 	return clapack.Dgels(trans, m, n, nrhs, a, lda, b, ldb)
 }
 
+// Dgesvd computes the singular value decomposition of the input matrix A.
+//
+// The singular value decomposition is
+//  A = U * Sigma * V^T
+// where Sigma is an m×n diagonal matrix containing the singular values of A,
+// U is an m×m orthogonal matrix and V is an n×n orthogonal matrix. The first
+// min(m,n) columns of U and V are the left and right singular vectors of A
+// respectively.
+//
+// jobU and jobVT are options for computing the singular vectors. The behavior
+// is as follows
+//  jobU == lapack.SVDAll		All M columns of U are returned in u
```

Comment:
Here and on the next three lines use spaces for the alignment of the second column.